packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/template.success.js

```diff
@@ -136,24 +119,42 @@ const Success = props => {
     )
   } else if (type === 'sell') {
     return (
-      <CheckoutWrapper>
-        <ExchangeCheckout
-          fiatLimits
-          base={base}
-          quote={quote}
-          errors={errors}
-          fiat={'USD'}
-          crypto={'BTC'}
-          accounts={accounts}
-          onSubmit={onSubmit}
-          fetchQuote={fetchQuote}
-          limits={limits[type]}
-          showRequiredMsg={step !== 'verified'}
-          requiredMsg={<RequiredMsg step={step} type={type} />}
-          continueButton={<ContinueButton step={step} type={type} />}
-          reasonMsg={<ReasonMsg reason={reason} limit={profile.limits[type]} />}
-        />
-      </CheckoutWrapper>
+      <Stepper initialStep={0}>
+        <StepView step={0}>
+          <CheckoutWrapper>
+            <BuyCheckout
+              quoteR={sellQuoteR}
+              account={accounts[0]}
+              onFetchQuote={fetchSellQuote}
+              reason={reason}
+              finishAccountSetup={finishAccountSetup}
+              limits={limits.sell}
+              type={'sell'}
```

Comment:
Don't really love this naming, if we can share `BuyCheckout`, can we call it something more generic like `ExchangeCheckout`, and pass in `type`?